### ISSUE TITLE
Keyboard Focus with RDP 

### ISSUE THREAD

**Issue by popxunga** [asbru-cm/asbru-cm#638]
Hi Team !
While using an rdesktop session, if I click somewhere in the Asbru Connections area on the left, and then inside some Window in the rdesktop session, the keyboard focus remain at the Asbru Connections area. I have to press the "Set keyboard focus" at the bottom or switch to another tab and then back to the rdesktop session in order to get the keyboard focus back on the rdesktop session.
This is with latest loki build on EL 7.8.
Thanks.

**Comment by vibo69** [comment]
I can confirm this. I use same workaround. I am not sure but probably also with xfreerdp. maybe helpful to mention will be that I use 'mouse follow focusing'
Debian 10 + xfce, loki branch 6.3.0

**Comment by gfrenoy** [comment]
Mmm, I already looked into that issue (#148) but we still have the main root issue : I did not find any signal that would let me know when the RDP window (that is an "external" XWindow) has been clicked.

Still, I found out that I'm receiving a signal when the mouse is entering into that Window ; I used it to force the keyboard focus.

Just pushed the change on the loki branch, let me know if this helps.
